README.md

```diff
@@ -11,6 +14,7 @@ This is supposed to be a window that shows up during first boot into openSUSE, s
 </a>
 
 ## About
-openSUSE Welcome is a utility that welcomes users to openSUSE. It includes links to various resources.
+openSUSE Welcome is a utility that welcomes users to openSUSE. It includes links to various resources, 
+including a link to the donation page at https://geekos.org.
 
-![Screenshot of openSUSE Welcome.](https://github.com/openSUSE/openSUSE-welcome/raw/master/Welcome.png)
+![Screenshot of openSUSE Welcome.](https://github.com/patrickfitz/openSUSE-welcome/blob/master/Welcome.png)
```

Comment:
you probably want to replace patrickfitz with openSUSE (I understand why you did it to see the actual change).